v5/owner.apib

```diff
@@ -0,0 +1,449 @@
+
++ Response 201 (application/json)
+    + Attributes (Times)
+
++ Response 404
+Returned if parking space or period was not found or if the parking space doesn't belong to the user.
+    + Body
+
++ Response 422
+Returned if weekday is not in range 1-7, weekday already exists, time is invalid, weekday or times are missing, time is overlapping or an error occurred.
+    + Attributes (Api Error)
+
+### Update Times [PUT /owner/parking-spaces/{parking_space_id}/periods/{period_id}/times/{times_id}]
+Updates the times by setting the values of the parameters passed.
+
++ Parameters
+    + parking_space_id : `00cd7cfd-e42d-11e2-8bf1-8a83f3373875` (string, required) - the parking space id
+    + period_id : `00cd7cfd-e42d-11e2-8bf1-8a83f3373875` (string, required) - the parking space id
+    + times_id : `00cd7cfd-e42d-11e2-8bf1-8a83f3373875` (string, required) - the parking space id
+
++ Request  (application/json)
+    + Attributes
+        + `time_start`: `2017-12-15` (string) - can be null
+        + `time_end`: `null` (string) - can be null
+
+    + Headers
+
+            Authorization: Basic MDEyMzQ1Njc4YWJjZGVmOnBhcmt1
+
++ Response 200
+    + Attributes (Times)
+
++ Response 404
+Returned if parking space, period or time was not found or if the parking space doesn't belong to the user.
+    + Body
+
++ Response 422
+Returned if weekday is not in range 1-7, weekday already exists, time is invalid, weekday or times are missing, time is overlapping or an error occurred.
+    + Attributes (Api Error)
+
+### Delete a Times [DELETE /owner/parking-spaces/{parking_space_id}/periods/{period_id}/times/{times_id}]
+
++ Parameters
+    + parking_space_id: `00cd7cfd-e42d-11e2-8bf1-8a83f3373875` (string, required) - the parking space id
+    + period_id: `00cd7cfd-e42d-11e2-8bf1-8a83f3373875` (string, required) - the period id
+    + times_id : `00cd7cfd-e42d-11e2-8bf1-8a83f3373875` (string, required) - the parking space id
+
++ Request
+    + Headers
+
+            Authorization: Basic MDEyMzQ1Njc4YWJjZGVmOnBhcmt1
+
++ Response 200
+    + Attributes (Times)
+
++ Response 404
+Returned if parking space, period or time was not found or if the parking space doesn't belong to the user.
+    + Body
+
++ Response 422
+Returned if an error occurred.
+    + Body
+
+# Data Structures
+
+## Parking space (object)
++ `id`: `005c4826-4e28-11e3-a675-d43d7eece53d` (string)
++ `commission`: `33.33` (number, in percent)
++ `status`: `active` (string)
++ `currency`: `EUR` (string)
++ `code`: `ASDF12` (string)
++ `type`: `offstreet` (string)
++ `price` (Owner Price)
++ `options` (array) - describes which options are available at the location and can contains each of the following elements. The list can be **empty**.
+    + (enum)
+      + `excess length` - the parking space can be used by cars with a small trailer
+      + `indoor` - the parking space has a roof or can be an underground garagethe parking space has a roof or can be an underground garage
```

Comment:
ah I see, you can copy this options list for the other options list that I mentioned at the beginning or put the options array into a different type and use the Include syntax (example is at v5/location.apib at `Location Price`)

Reply:
I used the Include syntax here.